lightning_pose_app/ui/extract_frames.py

```diff
@@ -825,3 +903,221 @@ def _render_streamlit_fn(state: AppState):
 
             #     #force rerun to show "waiting for existing..." message
             st_autorefresh(interval=2000, key="refresh_extract_frames_after_submit")
+
+    # Custom CSS for styling
+    st.markdown(
```

Comment:
are all of these entries necessary? ok if they are, but this is a lot of code and we should only include the minimum required

Reply:
yes. streamlit needs long text to set all those viz parmeters